src/pages/FAQ/index.jsx

```diff
@@ -628,7 +628,7 @@ export const FAQ = () => {
               <li>
                 <Text className="mt10">
                   <FormattedMessage
-                    defaultMessage="Eth1 address withdrawal: If you want to withdraw to Eth1 chain after the merge, you can set {eth1AddressWithdraw} when running deposit-cli. {boldWarning}"
+                    defaultMessage="Eth1 address withdrawal: If you want to withdraw to PoW chain (Eth1) after the merge, you can set {eth1AddressWithdraw} when running deposit-cli. {boldWarning}"
```

Comment:
Note: `--eth1_withdrawal_address` is a spec parameter name. So, here I try to let users understand Eth1 address means PoW chain address.